### ISSUE TITLE
More meaningful errors when .env file is missing

### ISSUE THREAD

**Issue by dmitrybelyakov** [projectshift/shift-boiler#93]

**Comment by dmitrybelyakov** [comment]
When originally initialising a project, the `.env` file is generated from a template. But because it is also ignored, to prevent committing secrets to source control, userland app will fail to run if the user forget's to re-create this file upon subsequent instantiation. The error given is quite cryptic:
```
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```

**Comment by dmitrybelyakov** [comment]
Fixed for the 0.7.12 release
